content/auth/token.textile

```diff
@@ -408,6 +457,29 @@ signature_base64 = base64.urlsafe_b64encode(signature).decode('utf-8')
 ably_jwt = base64_header + "." + base64_claims + "." + signature_base64
 ```
 
+```[php]
+$header = [
+    'typ' => 'JWT',
+    'alg' => 'HS256',
+    'kid' => '{{API_KEY_NAME}}'
+];
+$currentTime = time();
+$claims = [
+    'iat' => $currentTime,
+    'exp' => $currentTime + 3600,
+    'x-ably-capability' => '{\"*\":[\"*\"]}'
+];
+$base64Header = base64_encode(json_encode($header));
+$base64Claims = base64_encode(json_encode($claims));
+$signature = hash_hmac(
+    'sha256',
+    $base64Header . '.' . $base64Claims,
+    '{{API_KEY_SECRET}}',
+    true
+);
+$jwt = $base64Header . '.' . $base64Claims . '.' . $signature;
+```
+
```

Comment:
A the end, you can add a line 
- You can refer to https://github.com/ably/laravel-broadcaster/blob/83cc6bf242f9d1533ce49dbf077820426f08b15e/src/AblyBroadcaster.php#L200 for detailed implementation.
- Above code hasn't changed since last 2 years, won't change in the future either. So, it can be a good reference point for actual implementation.

Reply:
Is this needed? We discuss the same information, on the page. Plus it's also covered in the other languages (some I've yet to add)

Reply:
Okay, I think it's fine if we don't add it for now. 
I will take this is working if you have implemented and tested it in a separate repository